package.json

```diff
@@ -131,6 +132,7 @@
     "babel-eslint": "^10.0.3",
     "chokidar": "^3.3.0",
     "concurrently": "^5.0.0",
+    "conventional-changelog-cli": "^2.0.31",
```

Comment:
Added this for changelog generation because of this  issue https://github.com/conventional-changelog/standard-version/issues/509, which is currently open